projects/demo/src/components/control/index.html

```diff
@@ -0,0 +1,36 @@
+<tr>
+    <td colspan="3">
+        <span tuiTitle>
+            <strong>ControlOptions</strong>
+            <span tuiSubtitle>Applied as a host directive</span>
+        </span>
+    </td>
+</tr>
+
+<tr
+    *ngIf="!hiddenOptions.includes('readonly')"
+    name="[readonly]"
+    tuiDocAPIItem
+    type="boolean"
+    [(value)]="readonly"
+>
+    Readonly state
+</tr>
+<tr
+    *ngIf="!hiddenOptions.includes('disabled')"
+    name="[disabled]"
+    tuiDocAPIItem
+    type="boolean"
+    [(value)]="disabled"
+>
+    Disabled state
+</tr>
+<tr
+    *ngIf="!hiddenOptions.includes('invalid')"
+    name="[invalid]"
+    tuiDocAPIItem
+    type="boolean"
```

Comment:
It's `boolean | null` so we can manually set `true`/`false` invalid state or leave to the native behavior using `null`.

Reply:
Type updated

Reply:
I noticed change detection is broken there, when I set it to `true` it only updates when I click away to trigger change detection.

Reply:
And [readOnly] too

Reply:
Created #9389 9389